app/javascript/Search/Search.jsx

```diff
@@ -14,33 +9,23 @@ const GLOBAL_SEARCH_KEY = 'Slash';
 const ENTER_KEY = 'Enter';
 
 export class Search extends Component {
-  static defaultProps = {
-    searchBoxId: 'nav-search',
-  };
-
   constructor(props) {
     super(props);
     this.enableSearchPageChecker = true;
     this.syncSearchUrlWithInput = this.syncSearchUrlWithInput.bind(this);
+    this.searchInputRef = createRef(null);
```

Comment:
Using a [ref](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs) now so that we do not need to rely on an ID for the search form.